### DISCUSSION TITLE
Community themes

### DISCUSSION THREAD

**Discussion by Taiko2k** [Taiko2k/Tauon#461] [Show and tell]
Feel free to use this thread to share themes with other people. Be sure to include a screenshot. You can attach a zip file of the theme file to the post.

Remember, you can install themes by placing the `.ttheme` files in the `theme` subfolder of the *user data folder*. (You can find the *user data folder* by going MENU > Settings... > "Open data folder".)

If you want to create your own theme, see [here](https://github.com/Taiko2k/TauonMusicBox/wiki/Theming) for instructions.

**Comment by ghost** [comment]
# Stargazing
![screen04](https://user-images.githubusercontent.com/70255109/107908590-50489b80-6f4e-11eb-84ad-56e61c767863.png)
My personal theme for my Arch Linux rice.
https://github.com/VEGEMITEGOD/Tauon-Music-Box-theme-Stargazing/blob/main/stargazing.ttheme

**Comment by redsolver** [comment]
# Dracula Theme

Download: https://gist.github.com/redsolver/9a0fbf5d345bb2734b03b1419d620aa5

![DraculaTauon](https://user-images.githubusercontent.com/30355444/116786553-8f8f7e80-aa9f-11eb-9898-8dae10ad68f2.png)

**Comment by SirGorak** [comment]
It's a great player, but it's missing light themes. So I made one.
https://github.com/SirGorak/theme/blob/main/Light%20Candy.ttheme
![Screenshot_20210503_161223](https://user-images.githubusercontent.com/68249273/116887292-70712800-ac2a-11eb-9183-36be5fbe5171.jpg)

**Comment by keystroke3** [comment]
# Primera
![gallery_with_columns](https://user-images.githubusercontent.com/37796862/116885757-ebd5d800-ac30-11eb-9ee5-e6b537268ce1.png)
https://github.com/keystroke3/Tauon-themes#Primera

**Comment by aleister888** [comment]
Breeze (Dark), Gruvbox and Solarized themes: [themes.tar.gz](https://github.com/aleister888/artixRC-dotfiles/raw/refs/heads/main/assets/themes.tar.gz)
![Screenshot](https://github.com/user-attachments/assets/09f2ed06-3b6f-4fb2-8874-de9e8bafa90f)

**Comment by SirGorak** [comment]
Another version with blue accent
https://github.com/SirGorak/theme/blob/main/Light%20Sky.ttheme
![Screenshot_20210512_205335](https://user-images.githubusercontent.com/68249273/118029647-a61dbb80-b364-11eb-9960-251db49e594a.png)

**Comment by ggttttggggtt** [comment]
### Groove
https://github.com/ggttttggggtt/tauon-gruvbox
![image](https://user-images.githubusercontent.com/87790164/146628437-830b477c-5516-482c-902b-b82264b1dbc5.png)



Soft dark theme using the gruvbox color palette.

**Comment by phoenix492** [comment]
### OneDark Cool
https://github.com/Pieman492/OneDarkCool
![image](https://user-images.githubusercontent.com/17846991/132450429-8f62ee89-2611-46bb-8189-e0584e76d3ea.png)
A simple, flatish theme based on the cool, blueish colors from the OneDark palette.

**Comment by coral-hungus** [comment]
### Nord

I made a quick and dirty Nord theme - enjoy!
![Screenshot from 2022-02-12 12-50-18](https://user-images.githubusercontent.com/86207274/153724434-5b2d80ea-0e30-402f-beab-3b638888fbaf.png)
[nord.ttheme.zip](https://github.com/Taiko2k/TauonMusicBox/files/8054303/nord.ttheme.zip)

**Reply by rexendevar** [reply]
very excited to see yob in the wild as they are my favorite metal band :) also very cool theme

**Comment by aandmsantos0910** [comment]
### **Nord-Mod-Dark**

![TMB Nord Mod Dark Theme Preview](https://user-images.githubusercontent.com/119745242/205472599-6ca5ed02-c430-4ce4-a0a7-02dfb3c2b737.png)
[nord-mod-dark.zip](https://github.com/Taiko2k/TauonMusicBox/files/10147594/nord-mod-dark.zip)

**Comment by 1tituz** [comment]
endeavourOS Theme.
Why not?
![endeavourOS](https://user-images.githubusercontent.com/101503597/213863164-220a6ea4-31c6-43f1-9a69-3b6349f0ec2d.png)

[endeavourOS.ttheme.zip](https://github.com/Taiko2k/TauonMusicBox/files/10471980/endeavourOS.ttheme.zip)

**Reply by Taiko2k** [reply]
Wow, nice

**Reply by Solarunit** [reply]
I also like how you're a heavy queue user as me :)

**Reply by 1tituz** [reply]
queing is essential ;)

**Comment by Achardir** [comment]
[Catppuccin](https://github.com/Achardir/catppuccin-tauon)

![](https://raw.githubusercontent.com/Achardir/catppuccin-tauon/main/Catppuccin_Tauon.png)

**Comment by 1tituz** [comment]
80s Dark Theme. 

![80s_Dark](https://github.com/Taiko2k/TauonMusicBox/assets/101503597/69692389-b53b-4775-93a5-832a01693c76)
![80s_Dark_2](https://github.com/Taiko2k/TauonMusicBox/assets/101503597/a6c667f5-5c14-4368-8178-b69269b6d705)
[80s_Dark.ttheme.zip](https://github.com/Taiko2k/TauonMusicBox/files/13203996/80s_Dark.ttheme.zip)

**Comment by lukyy22** [comment]
tried to create a ayu mirage theme 😄 

![ayu-mirage-tmb](https://github.com/Taiko2k/TauonMusicBox/assets/97752378/b2c4c5be-20dc-489a-bd09-9c30ece6e9f4)

[tauonmb-ayu-mirage.7zip](https://github.com/lukyy22/configs/raw/main/tauonmb-ayu-mirage.7z)

**Comment by make-42** [comment]
# Pywal theme (auto updates with wallpaper)
![image](https://github.com/user-attachments/assets/82e8f047-999c-47a5-bb37-9e2855d96df6)
![image](https://github.com/user-attachments/assets/ed89fa1d-c4ab-409b-baf8-e8872c6b14a1)

## Disclaimer
This only works on Linux (or maybe Mac (I haven't tested it)) since it requires `pywal`.

**As of the latest version release, Tauon doesn't recognize links. A fix has been pushed but you have to wait till the next version releases to use this theme.**

## How to install
 - Add the `colors-tauon.ttheme` file to your `~/.config/wal/templates/` directory.
 - Re-run pywal to regenerate the theme file.
 - Create a link to the theme file with `ln -s $HOME/.cache/wal/colors-tauon.ttheme $HOME/.local/share/TauonMusicBox/theme/pywal-tauon.ttheme`
 
 ## Notes
 You will need to restart Tauon whenever pywal is run to update the colors.
 
 ## Download
[colors-tauon.ttheme.zip](https://github.com/user-attachments/files/16482486/colors-tauon.ttheme.zip)

**Reply by rexendevar** [reply]
I've set this up and I have it creating the theme file correctly and linking the theme file correctly, but when I go into the settings for Tauon, that particular theme is not there in the list. What am I doing wrong?

**Reply by rexendevar** [reply]
i corrected the last command to say `ttheme` instead of `theme` and made the path point to the correct place for the flatpak. I also removed all the `.rgb` in the template file. the 80s theme a couple posts up loads in fine but not this.

**Reply by make-42** [reply]
Oh yeah sorry for that typo. It doesn't work since Tauon doesn't recognize links yet unless you are running it at a version after this ( 7e1e2a6) commit (I asked Taiko to push a fix so that Tauon recognizes links). If you wait till the next version release, you'll probably be able to use it then. I should've specified this in the initial comment. Sorry.

You can still use the theme by manually copying the generated theme file but you'll have to do so everytime your colors change.

(And also due to the nature of how the links are resolved by Tauon after that fix, the extension of the link isn't what is recognized by Tauon but the extension of the target file)

**Comment by Zutyosh** [comment]
# Primerya

Primerya is a remix of @keystroke3's Primera theme with full support for transparent accents!

![Copie d'écran_20250223_124950](https://github.com/user-attachments/assets/70c1fd62-acaa-40cf-a00e-ff64487f9385)
![Copie d'écran_20250223_125024](https://github.com/user-attachments/assets/dfc85563-eb8f-4610-b4fa-7b12e607b59e)
![Copie d'écran_20250223_125228](https://github.com/user-attachments/assets/c0d74a0e-7835-4ea5-9639-8471afed09a5)


# Download
[Primerya.ttheme.zip](https://github.com/user-attachments/files/18930173/Primerya.ttheme.zip)

**Reply by keystroke3** [reply]
I am happy to see people  still use and improve the theme. Looks nice.

**Reply by Zutyosh** [reply]
Yes, Primera is my favourite and main theme! But I find it a bit sad that it doesn't take advantage of the new transparency option.

**Reply by Taiko2k** [reply]
@TheFlomax Its not currently intended that themes apply their own transparency. (Though it is possible by setting a 4th alpha value in theme colours, results may be unpredictable)

**Comment by OtherBreathingYuria** [comment]
## A Green Theme
[fukujusou.ttheme.zip](https://github.com/user-attachments/files/19018388/fukujusou.ttheme.zip)
![Screenshot from 2025-02-27 17-36-49](https://github.com/user-attachments/assets/15abccfc-ddfe-4adf-875b-5a515676dac3)

**Comment by kreetzel** [comment]
[Everforest](https://github.com/kreetzel/Pretzels-Everforest-themes/tree/main/tauon)
![Screenshot](https://github.com/user-attachments/assets/1c885411-87a6-4629-933f-2024f72cefd1)

**Comment by ninepoint-eight** [comment]
Made some themes to blend in with the default Libadwaita theme on GNOME

https://codeberg.org/nine_point_eight/tauon-themes

![adwaita-dark-blue](https://github.com/user-attachments/assets/9dfc1a8d-0e76-42a8-b607-bf0a68bc91bb)
![adwaita-dark-red](https://github.com/user-attachments/assets/44cb0482-41e4-4370-9aab-d5617fa3831a)
![adwaita-light-green](https://github.com/user-attachments/assets/d852647b-3d17-4192-9270-7dc2154c5e3b)

**Comment by rexendevar** [comment]
Now that my pull request is merged, I can share my version of the Sunken theme. It is now possible to make the lyrics panel transparent and to manually color the lyrics text.
[Sunken Translucent.txt](https://github.com/user-attachments/files/20826613/Sunken.Translucent.txt)
![image](https://github.com/user-attachments/assets/c6301974-28f2-4730-8c93-32eb4c818543)
DISCLAIMER: Taiko is right about results being unpredictable with transparency. Due to the way the text is antialiased, I strongly recommend you only use this theme if your wallpaper is blue. Also remember to rename the downloaded file to a `.ttheme`

**Comment by wixdaq** [comment]
### Tokyo Night
https://github.com/wixdaq/theme
![Screenshot_20250626_123642](https://github.com/user-attachments/assets/dcd36dc8-82cd-44f9-bd24-2fa7cc06db23)

**Reply by Solarunit** [reply]
very nice!

**Reply by wixdaq** [reply]
Thanks :)

**Comment by kreetzel** [comment]
Everblush
[everblush.txt](https://github.com/user-attachments/files/21074595/everblush.txt)
![Screenshot_20250705_114457](https://github.com/user-attachments/assets/03318367-037b-4cc0-8d76-092ebc49b59b)

**Comment by val-byte** [comment]
## Oxocarbon

Oxocarbon is a High contrast accessible colorscheme inspired by IBM Carbon. It delivers class-leading readability without strain by adhering to WCAG 2.1 guidelines

[Oxocarbon Tauon-MB](https://github.com/val-byte/Oxocarbon-Tauon-MB)

<img width="1277" height="710" alt="image" src="https://github.com/user-attachments/assets/7179fa02-c3aa-4079-884b-f7ce787c91d6" />

**Reply by MsFontaine** [reply]
Love this theme, thanks!

**Comment by udoaarts** [comment]
## Mint-X-Grey
A monochrome theme made to match the Linux Mint X Grey light theme.

<img width="1920" height="1031" alt="Screenshot_2025-08-17_11-58-00" src="https://github.com/user-attachments/assets/0ab7ac53-27ee-4ba0-8201-6c15d0d8e741" />


[Mint-X-Grey.ttheme.zip](https://github.com/user-attachments/files/21821588/Mint-X-Grey.ttheme.zip)

**Comment by DanteDragan** [comment]
### Crimson Sparta
---
Black, white and crimson for ones, who want brutality.

<img width="2530" height="1325" alt="image" src="https://github.com/user-attachments/assets/e4994726-78d7-4431-97e9-51295cbcbb67" />

[tauon_sparta-theme.zip](https://github.com/user-attachments/files/21948643/tauon_sparta-theme.zip)

**Comment by CJTS15** [comment]
### Dracula Purple Theme

[Dracula.ttheme.tar.gz](https://github.com/user-attachments/files/22245502/Dracula.ttheme.tar.gz)

<img width="1741" height="962" alt="2025-09-10_09-02" src="https://github.com/user-attachments/assets/43742ea9-3909-4833-abdf-02f847516984" />

**Comment by timofey260** [comment]
## [Solarized Dark](https://ethanschoonover.com/solarized/) Magenta
[Solarized-Dark.ttheme.tar.gz](https://github.com/user-attachments/files/24887636/Solarized-Dark.ttheme.tar.gz)
<img width="1919" height="1051" alt="image" src="https://github.com/user-attachments/assets/1f42ef30-8d49-4a98-9748-83ef19b0be31" />
<img width="1916" height="1050" alt="image" src="https://github.com/user-attachments/assets/29e455ef-4ea1-4f1f-99ed-a84fd22018d5" />
<img width="1919" height="1052" alt="image" src="https://github.com/user-attachments/assets/ea5b4f87-caee-4799-8241-fabfefbe819a" />

**Comment by nicgil23** [comment]
# Catppuccin Mocha
<img width="1920" height="1200" alt="2026-05-14-185812_hyprshot" src="https://github.com/user-attachments/assets/7fff5602-5870-40e0-857f-311a8a980b34" />
While Tauon does have a native transparency setting, the one shown here is handled via my Hyprland window rules.

[CatppuccinMocha.txt](https://github.com/user-attachments/files/27769720/CatppuccinMocha.txt)

**Comment by WasabiIceCream** [comment]
# Noctalia  
  
This is a template I made for [Noctalia Shell](https://noctalia.dev/).  
It will auto theme Tauon's colors to match Noctalia's, either from its selected color scheme or auto-generated from your wallpaper.  
I used @Zutyosh's Primerya as a starting point, which was originally based on  @keystroke3's Primera.  
Edited to support full transparency, so you can see your wallpaper through the backgrounds.  
  
  
EDIT: This theme is now included as a community template in Noctalia v5.  
You can enable it in the Noctalia Settings - Templates tab - and Enable Community Templates.  
Select the "tauon" template from here. You can then select the "Noctalia" theme in the Tauon theme settings.  
If anyone thinks the theme can be improved, you can suggest those changes [HERE](https://github.com/noctalia-dev/community-templates).  
  
  
The theme should automatically show up and auto-update in Tauon whenever your wallpaper changes.  
This'll only work on whatever platform Noctalia Shell would work, obviously. XD  
Only "problem" is that Tauon does not live-update changes in the theme for the UI,  
so you'll have to manually press the F10 key to refresh the UI for changes.  
Hopefully we can get it to auto-refresh the UI soon @Taiko2k _wink wink_  
  
  
Dark Mode:  
<img width="3840" height="2160" alt="Screenshot from 2026-05-15 23-41-13" src="https://github.com/user-attachments/assets/99fb488a-a570-45c6-a6e5-b8fb74d5bff0" />  
<img width="3840" height="2160" alt="Screenshot from 2026-05-16 00-15-27" src="https://github.com/user-attachments/assets/471e5d2c-db0f-4293-8e2d-49e4728aa2e4" />  
  
Light Mode:  
<img width="3840" height="2160" alt="Screenshot from 2026-05-16 18-29-13" src="https://github.com/user-attachments/assets/38f64c08-d233-4af2-b4b6-8fd70e8e0d58" />  
<img width="3840" height="2160" alt="Screenshot from 2026-05-16 18-31-57" src="https://github.com/user-attachments/assets/c5acae7a-1f4b-45fb-a60f-37925957a40e" />

**Reply by Zutyosh** [reply]
I don't use Noctalia Shell but is look wonderful I like it, good job !

**Comment by notkiniro** [comment]
### Simple Monochromatic Theme
A simple Tauon Music Box theme consisting of black and white accents. 

Download Here:
[black_and_white.ttheme.zip](https://github.com/user-attachments/files/27871425/black_and_white.ttheme.zip)

### How to Install
On Tauon, Go to 
_Menu_ > _Settings_ 
Click on the _Open data folder_ button located inside _Function_ section
Drop the **.ttheme** file inside the _theme_ directory
Restart Tauon Music Box and select the theme

### Preview
<img width="1425" height="794" alt="black_and_white_preview" src="https://github.com/user-attachments/assets/1edf8858-e469-42ab-bf60-9a713c69b2be" />

**Reply by racoonez** [reply]
So simple but also nice, also instead of a plain white you can use also the Azure blue ---> #F0FFFF the blue tint in it is very very subtle but yet makes a cool contrast. 😃

**Comment by 1tituz** [comment]
### Ember
**A glowing ember remix version of the Ash theme.**

Essentially, it’s the already beautiful Ash Theme, enhanced with a few touches to better highlight the elements that matter to me.
Perhaps someone will find it useful...
Download: [Ember.zip](https://github.com/user-attachments/files/28824522/Ember.zip)

<img width="2560" height="1389" alt="screen1" src="https://github.com/user-attachments/assets/e954d085-18ed-4a27-86b3-f3ada4c98aab" />
<img width="2560" height="1389" alt="screen2" src="https://github.com/user-attachments/assets/c0d71714-b545-4725-bdea-1679e3aaacb3" />
<img width="2560" height="1389" alt="screen3" src="https://github.com/user-attachments/assets/e73218f9-8ab4-4f2d-9dff-3c4f7be82c6a" />

**Reply by Taiko2k** [reply]
Would you be interested in allowing us to replace the official Ash theme in Tauon with your improved Ember version?

**Reply by 1tituz** [reply]
Of course! It would be an honour! :)

**Comment by albibireal** [comment]
**Beige & Taupe**

A simple transparent theme with a taupe background and beige text & buttons.

[Beige&Taupe.zip](https://github.com/user-attachments/files/30173638/Beige.Taupe.zip)
<img width="1917" height="1080" alt="Beige TaupeImage1" src="https://github.com/user-attachments/assets/76550698-1714-41a6-831e-26aebf967c42" />
<img width="1919" height="1080" alt="Beige TaupeImage2" src="https://github.com/user-attachments/assets/c9ab829e-fcb8-4292-a694-a6b72312e64b" />

**Comment by Neptune3013** [comment]
# Sweet Red Berry

Theme with Red + Purple color palette with moderately contrasting colors.

[Sweet Red Berry.zip](https://github.com/user-attachments/files/31328312/Sweet.Red.Berry.zip)



<img width="1920" height="1080" alt="Screenshot_20260822_110216" src="https://github.com/user-attachments/assets/c516e4bc-b514-4103-8f0b-6b5b561ff2a7" />
<img width="1920" height="1080" alt="Screenshot_20260822_103421" src="https://github.com/user-attachments/assets/f4d57a93-c65d-4120-ab60-042a056ff544" />
<img width="1920" height="1080" alt="Screenshot_20260822_103022" src="https://github.com/user-attachments/assets/82cb3be7-9000-4e23-a57c-7b56674b47af" />

# Hi Con Red Berry

Similar color palette with high contrast colors.
[Hi Con Red Berry.zip](https://github.com/user-attachments/files/31328314/Hi.Con.Red.Berry.zip)


<img width="1920" height="1080" alt="Screenshot_20260822_111253" src="https://github.com/user-attachments/assets/ef60943b-67c8-46f9-84ea-d335aa673085" />
<img width="1920" height="1080" alt="Screenshot_20260822_111216" src="https://github.com/user-attachments/assets/e333df27-ec6a-43b7-a552-4e9ede6cb451" />
